renovate.json5

```
@@ -20,6 +20,13 @@
         "/^fumadocs-ui$/"
       ]
     },
+    {
+      "groupName": "trigger.dev",
+      "matchPackageNames": [
+        "/^@trigger\\.dev/",
```

suggestion: a slash needed?

```suggestion
        "/^@trigger\\.dev\//",
```